source/nodejs/adaptivecards-designer/src/card-designer.ts

```diff
@@ -705,12 +705,12 @@ export class CardDesigner extends Designer.DesignContext {
                                 thumbnail: () => {
                                     const thumbnail = document.createElement("div");
                                     thumbnail.className = "acd-image-container";
-                                    thumbnail.innerHTML = `<div class="acd-image-upload acd-open-sample-item">
+                                    Utils.setInnerHtml(thumbnail, `<div class="acd-image-upload acd-open-sample-item">
```

Comment:
It looks like we only use dynamic templated strings for the privacy policy link in `open-image-dialog.ts`. If possible, that should be moved over to using a static string. Everywhere else we're calling `setInnerHtml()`, we should use non-templated strings (i.e. move from using `` ` `` to using `'` + `+` to build the strings). Once that's done, `ttPolicy` should get a comment stating that it's only intended for use in hardcoded content strings.

Reply:
The correct way to address this is that we build the <div> from code. There are two solutions here:
1. I create a task to remove Utils.setInnerHtml
2. or I revert the change on adaptivecards-designer.
which way do you want to go

Reply:
I revert the change to adaptivecards-designer

Reply:
issue is tracked by #7842